packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -584,10 +599,44 @@ export class CfnInclude extends core.CfnElement {
     return cfnCondition;
   }
 
+  private getPlaceholderID(): string {
```

Comment:
Why not a uuid or a counted string? `PlaceholderLogicalId${this.counter++}` ?

Reply:
Importing uuid would add another dependency to the package, so I avoided it to keep the change small.
Counted string would work, don't feel strongly though either way. Do you have a strong preference?

Reply:
Counted string works.

Reply:
And yet I still see the random string generating code in the PR ?

Reply:
https://github.com/aws/aws-cdk/pull/22126/commits/8b1fb3c03256a182c975891b72fd7c7da5729585#diff-4ab381cb44660afe3682e0220f1605878c1658f6a8012d5c1ea8bf51e2d4d3dfR603 It shows up with the random placeholder generation removed and counting by map size added :)